internal/handlers/jsonb1/where.go

```diff
@@ -142,6 +142,17 @@ func fieldExpr(field string, expr *types.Document, p *pg.Placeholder) (sql strin
 			// {field: {$gte: value}}
 			sql += "_jsonb->" + p.Next() + " >="
 			argSql, arg, err = scalar(value, p)
+		case "$size":
+			// { field: { $size: 2 } }
+			var sizeValue any
+			sizeValue, err = expr.Get(op)
+			if err = common.IsPositiveInteger(sizeValue); err != nil {
+				err = common.NewError(common.ErrBadValue, err)
+				return
+			}
+
+			sql += "jsonb_typeof(_jsonb->" + p.Next() + ") = 'array' AND jsonb_array_length(_jsonb->" + p.Next() + ") = "
```

Comment:
That should be the same placeholder (number) to avoid sending the same argument (field name) twice